docs/source/docker/getting_started/first_steps.rst

```diff
@@ -621,13 +621,13 @@ The output directory is structured in the following way:
 * data:
    The data directory contains everything to do with data. 
    
-    * If `enable_corruptness_check=True`, it will contain a "clean" version of the dataset. 
-    * If `remove_exact_duplicates=True`, it will contain a copy of the `embeddings.csv` 
-      where all duplicates are removed. Otherwise, it will simply store the 
-      embeddings computed by the model.
-    * If `selected_sequence_length > 1`, it will contain a `sequence_information.json`
+    * `embeddings.csv` contains all computed embeddings for input samples.
+    * `selected_embeddings_including_datapool.csv` contains the embeddings of all selected samples.
+    * If `enable_corruptness_check=True`, `data` will contain a "clean" version of the dataset. 
```

Comment:
Suggestion for improvement: "clean" is vague. What about this?
```
`<insert the name of subdir>` will contain a copy of input dataset with corrupt images removed
```
(Also I'm not sure if images is the right term here - I'm not sure how exactly we represent datasets, whether it is images / videos / a .txt file with s3 paths / etc.)

Reply:
For context: Corruptness check makes sure that all samples exist (if you specify the relevant filenames), can be accessed (read permissions are set), opened as images or videos (e.g. .jpg images should actually have the .jpg format) and don't contain image artefacts (e.g. many repeating stripes or being completely black or white.)
Perhaps you can add such kind of information @shaundaley39 .

I know that this is many different stuff that is checked for, see https://linear.app/lightly/issue/LIG-1646/docker-error-message-figure-out-how-to-disentangle-corruptness-check as issue for improving all of that.